src/main/resources/com/puppycrawl/tools/checkstyle/messages.properties

```diff
@@ -1,4 +1,10 @@
+#Checker.destroy=Unable to persist cache file.
```

Comment:
Please remove all commented lines and add them as part of a new PR when you work on it.

Reply:
Done..
Archived on local and removed for remote branch.